packages/api/cms-api/src/access-log/access-log.interceptor.ts

```diff
@@ -0,0 +1,59 @@
+import { CallHandler, ExecutionContext, Injectable, Logger, NestInterceptor } from "@nestjs/common";
+import { GqlExecutionContext } from "@nestjs/graphql";
+
+/** TODOs
+ * Docs are missing
+ * Api password check
+ */
+const DAM_IMAGE_PATH = "/dam/images/:hash/:fileId";
```

Comment:
Better:
`const ignoredPaths = ["/dam/images/:hash/:fileId", "/dam/files/:hash/:fileId"];`

Reply:
You could also prepare something like `ignoredResolvers` to exclude common graphql-operations (like currentUser)